UefiCpuPkg/Library/MpInitLib/DxeMpInitLib.inf

```diff
@@ -62,6 +62,7 @@
 [Protocols]
   gEfiTimerArchProtocolGuid                     ## SOMETIMES_CONSUMES
   gEfiMemoryAttributeProtocolGuid               ## CONSUMES ## MU_CHANGE
+  gCpuMpDebugProtocolGuid                       ## CONSUMES ## MU_CHANGE

```

Comment:
This library produces the protocol, right?

Reply:
Fixed